sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,18 +1,16 @@
 # Release History
 
+## 1.0.0-beta.5 (Unreleased)
+
 ## 1.0.0-beta.4 (2021-01-28)
 ### Breaking Changes
 - Renamed `CommunicationUserCredential` to `CommunicationTokenCredential`
 - Renamed `PhoneNumber` to `PhoneNumberIdentifier`
 - Renamed `CommunicationUser` to `CommunicationUserIdentifier `
 - Renamed `CallingApplication` to `CallingApplicationIdentifier`
 
-### Added
```

Comment:
What is the reason for removing the skipped versions here? 
Also according to the CHANGELOG guide, we want to use `Added` instead of other variations. 
https://azure.github.io/azure-sdk/policies_releases.html#changelog-guidance